pig/src/main/java/org/jboss/pnc/bacon/pig/Pig.java

```diff
@@ -253,6 +253,12 @@ public static class Run extends PigCommand<PigRunOutput> {
                 description = "Path to file with license names for license generation. Usually rh-license-names.json.")
         private String licenseNamesPath;
 
+        @Option(
+                names = "--oldBCNaming",
+                defaultValue = "false",
+                description = "if set to true names used in source generation will be set in old way using BC names")
+        private boolean oldBCNaming;
```

Comment:
Do we want to have the option here in general pig?

Reply:
People usually run `pig run` to do all steps in pipeline, that's why it's here. Now if you're talking generally if we want switch and not just option for people to put it in build-config.yaml I would say yes because people will complain that they need to do code change in their files.